### ISSUE TITLE
AI Lab Release + Testing 1.2.0, 1.2.1, 1.2.2, 1.2.3

### ISSUE THREAD

**Issue by nichjones1** [containers/podman-desktop-extension-ai-lab#1493]

**Comment by ScrewTSW** [comment]
The testing has concluded:
* [1.2.0](https://docs.google.com/spreadsheets/d/1TuJWb7c1tSbh-4ZPoYESAviDZiEhS15CmGb_Ky_Z1Cw/edit?gid=1122849411#gid=1122849411) 
* [1.2.1](https://docs.google.com/spreadsheets/d/1TuJWb7c1tSbh-4ZPoYESAviDZiEhS15CmGb_Ky_Z1Cw/edit?gid=800155737#gid=800155737)
* [1.2.2](https://docs.google.com/spreadsheets/d/1TuJWb7c1tSbh-4ZPoYESAviDZiEhS15CmGb_Ky_Z1Cw/edit?gid=433654580#gid=433654580)
* [1.2.3](https://docs.google.com/spreadsheets/d/1TuJWb7c1tSbh-4ZPoYESAviDZiEhS15CmGb_Ky_Z1Cw/edit?gid=857117216#gid=857117216)

Issues fixed during testing:
* https://github.com/containers/podman-desktop-extension-ai-lab/issues/1548
* https://github.com/containers/podman-desktop-extension-ai-lab/issues/385

New issues:
* https://github.com/containers/ai-lab-recipes/issues/762


In conclusion, the entire testing has passed in green on all platforms!
